src/Entity/Server.php

```diff
@@ -123,7 +131,10 @@ public function id() {
    */
   public function executeOperation(OperationParams $operation) {
     $previous = Executor::getImplementationFactory();
-    Executor::setImplementationFactory([\Drupal::service('graphql.executor'), 'create']);
+    Executor::setImplementationFactory([
```

Comment:
Please try to not reformat any code. It makes the PR harder to review. Can you remove the bits that were reformatted?